package.json

```diff
@@ -7,14 +7,15 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "express-body-parser-error-handler": "^1.0.4",
```

Comment:
We don't need this "express-body-parser-error-handler". Could you please delete it? `npm uninstall express-body-parser-error-handler`